### PR TITLE
Add hwaddr, macaddr, slaac filter plugin

### PR DESCRIPTION
**Pull request by ashwini-mhatre**
##### SUMMARY

Added hwaddr, macaddr, slaac filter plugin

##### ISSUE TYPE
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: changelogs/fragments/add_hwaddr_filter.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - Add hwaddr filter plugin.
```
Comment (Qalthos) [inline changelogs/fragments/add_hwaddr_filter.yaml:3]: Minor quibble, but these can all be in the same file if you're merging them at once
```suggestion
  - Add hwaddr filter plugin.
  - Add macaddr filter plugin.
  - Add slaac filter plugin.
```

**Review by Qalthos — COMMENTED** [review]
I get splitting out all the filter plugins to seperate files for maintainability, but macaddr is an alias for hwaddr, do we really need or even want to split them as well?

File: changelogs/fragments/add_hwaddr_filter.yaml (hunk 1 above)
Comment (ashwini-mhatre) [reply changelogs/fragments/add_hwaddr_filter.yaml]: ack

Comment (ashwini-mhatre) [reply changelogs/fragments/add_hwaddr_filter.yaml]: @Qalthos in the old ipaddr filters both hwaddr and macaddr mapped as different filter that's why created separate filters https://github.com/ansible-collections/ansible.netcommon/blob/main/plugins/filter/ipaddr.py#L1163

**Review by KB-perByte — APPROVED** [review]

**Pull request merged**